actors/market/src/lib.rs

```diff
@@ -966,27 +964,13 @@ impl Actor {
             }),
             None => {
                 // State::get_proposal will fail with USR_NOT_FOUND in either case.
```

Comment:
Change this comment to instead say we propagate the NOT_FOUND or EXPIRED from this call, and note that we can't distinguish between failing to activate, or having been cleaned up.

Reply:
Sure, Done